exercises/npm-registry/src/package.ts

```diff
@@ -1,22 +1,53 @@
-import { RequestHandler } from 'express';
-import got from 'got';
-import { NPMPackage } from './types';
+import { RequestHandler } from "express";
+import got from "got";
+import { minSatisfying } from "semver";
```

Comment:
You can use minVersion instead on minSatisfying so you can do the got request with the proper version

Reply:
You can use minVersion instead on minSatisfying so you can do the got request with the proper version